lib/spack/spack/test/cmd/find.py

```diff
@@ -98,3 +103,10 @@ def test_tag2_tag3(parser, specs):
     spack.cmd.find.find(parser, args)
 
     assert len(specs) == 0
+
+
+@pytest.mark.db
+@pytest.mark.usefixtures('database')
```

Comment:
Prefer to add this as a parameter than to use `@pytest.mark.usefixtures`

Reply:
Done